build.ps1

```diff
@@ -3,46 +3,29 @@
 [CmdletBinding(PositionalBinding = $false)]
 Param(
   [Parameter(ValueFromRemainingArguments = $true)][String[]]$verbs,
-  [Switch]$reset,
   [Switch]$prereqs
 )
 
+$upstream_repo_path = Join-Path -Path ".." -ChildPath "playwright"
```

Comment:
- Let's print this path as "Using upstream playwright at <PATH>"
- Let me override it with argument or env, and mention this in the ROLLING.md